nx.json

```diff
@@ -363,7 +363,8 @@
         "{projectRoot}/action.yml",
         "{projectRoot}/LICENSE"
       ],
-      "outputs": ["{projectRoot}/packaged-action"]
+      "outputs": ["{projectRoot}/packaged-action"],
+      "dependsOn": ["build"]
```

Comment:
is it really needed ?

Reply:
yes and I wonder why it was not the case before.
the `ncc` command build what is targeted by the `package.json`'s `main` field.
Currently, in our actions, this field is targeting the `tmp/index.js` which is the result of the `tsc` command triggered by the `build` (or `compile` and `github-action-compile`) task.